custom_components/homewhiz/select.py

```diff
@@ -17,31 +17,41 @@
 from .appliance_config import (
     ApplianceConfiguration,
     ApplianceFeature,
-    ApplianceFeatureBoundedOption,
     ApplianceFeatureEnumOption,
     ApplianceProgram,
 )
 from .config_flow import EntryData
 from .const import DOMAIN
-from .homewhiz import HomewhizCoordinator, brand_name_by_code
+from .homewhiz import HomewhizCoordinator, appliance_type_by_code, brand_name_by_code
 
 _LOGGER: logging.Logger = logging.getLogger(__package__)
 
 
+def unit_for_key(key: str):
+    if "TEMP" in key:
+        return " " + TEMP_CELSIUS
+    if "SPIN" in key:
+        return " RPM"
+    return ""
```

Comment:
This is different from the [implementation in sensor.py ](https://github.com/rowysock/home-assistant-HomeWhiz/blob/main/custom_components/homewhiz/sensor.py#L79) and also redundant.

Reply:
Done